recipes/robotraconteur/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    mkdir -p bluez/include/bluetooth
+    cp -r bluez-src/lib/*.h bluez/include/bluetooth/
+    CXXFLAGS="$CXXFLAGS -I$SRC_DIR/bluez/include"
```

Comment:
Without the export this is not visible to cmake. Maybe this is not needed at all?
```suggestion
    export CXXFLAGS="$CXXFLAGS -I$SRC_DIR/bluez/include"
```

Reply:
I am not sure why this compiled, since it does require the libbluetooth header files. It might have picked them up from the system path. I added `export`.